src/index.js

```diff
@@ -1,5 +1,9 @@
+const $ = require("jquery");
 import './styles.css';
 
-import { ArtistSearch } from './searches.js';
+import { Listener } from './listener.js';
 
-new ArtistSearch('queen').fetch_and_render();
+$(document).ready(function() {
+  $('#under-nav').height($('nav').height() + 20);
+  Listener.prototype.start();
```

Comment:
Do you have to use `Listener.prototype.start();` here or can you just use `Listener.start();`

Reply:
Since I'm calling it on the listener class, I have to use the prototype keyword. Otherwise I would have to instantiate and instance of listener.

Reply:
Gotcha, that makes sense. Thanks!